ext/json/ext/fbuffer/fbuffer.h

```diff
@@ -155,13 +157,14 @@ static long fltoa(long number, char *buf)
     if (sign < 0) number = -number;
     do *tmp++ = digits[number % 10]; while (number /= 10);
     if (sign < 0) *tmp++ = '-';
+    if (tmp > buf + FBUFFER_APPEND_LONG_BUFFER_SIZE) UNREACHABLE;
```

Comment:
This condition means that `tmp` overflowed.
Doesn't it result in another warning?

```suggestion
    if (tmp >= buf + FBUFFER_APPEND_LONG_BUFFER_SIZE) UNREACHABLE;
    if (sign < 0) *tmp++ = '-';
```